### PR TITLE
Calibr8 UI update

### LINKED ISSUE #51: Calibr8 UI is potentially confusing
image

Not sure the bug here, but Calibra8 overlaps calculate on latest

### PR DESCRIPTION
**Pull request by beau-seidon**
fixes #51 

the background Skyline visuals have been replaced with horizontal bars below the numbers in each section indicating cv input and output levels.

middle dividing line has been removed and other ui element spacing adjusted very slightly to clean everything up.

input and output meter bars can handle both positive and negative values:
if the value is positive, the corresponding bar extends from the left side of the segment toward the right as the value increases. 
if it is negative, the bar "wraps" like a pacman maze and goes from the right side toward the left as the value goes more negative.

the scale factor and offset parameters adjust the relative size and position of the bars.

### CODE REVIEW

**Comment by beau-seidon** [comment]
this is my first time working with O_C code, so if you have any preferences, "standards", or best practices you want me to follow, or have any recommended changes etc. please let me know. 

tl;dr if my code sux, tell me

**Comment by djphazer** [comment]
I love it! Code looks flawless, to be honest. It's just some math and stuff ;)
I wonder if maybe input should be top, output below it? I'll try it out myself, see how it feels. As long as there's a distinction between in and out, which there is.

**Comment by beau-seidon** [comment]
> I wonder if maybe input should be top, output below it? I'll try it out myself, see how it feels. 

just tested and i do like input on top better. fixt

**Comment by djphazer** [comment]
This looks good, so I'm gonna squash it and merge as a single commit.

I've cherry-picked the rest of your `midi-modz` branch stuff into my `dev/1.9` branch for testing...

PS: [Join the Discord](https://discord.gg/crxQGdX5)!

**Comment by beau-seidon** [comment]
sweet! thank you.

i was too slow. i just pushed a commit with a bunch more features to midi-modz. :)
UI update on hMIDIIn to allow longer mode names, and a fix for that activity icon on hMIDIOut.
some more note-out modes (the ones i mentioned the other day), and an inverse gate mode, and a couple new trigger modes. also i cleaned up the code i wrote a lot. 

gotta quit coding, someones gonna have to document all this stuff! (i'll do it...)
